packages/types/src/index.ts

```diff
@@ -316,7 +334,7 @@ export type Leg = {
   rentedBike: boolean;
   rentedCar: boolean;
   rentedVehicle: boolean;
-  route?: string;
+  route?: string | LegRoute;
```

Comment:
Is this for OTP1 support? Same as above, how do we feel about removing the string from this for 100% type compatibility with OTP2 GraphQL?